### PR TITLE
news.sky.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [X] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/135197

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -6870,6 +6870,7 @@ tmonews.com##.sponsor-dfp
 fc.lc###link-view > iframe[src*="/fc.lc/"]
 news.sky.com##.sdc-site-au[data-ad-format] > div[data-role="ad-container"]
 news.sky.com##.sdc-site-layout-sticky-region__ghost[data-format="leaderboard"]
+!+ NOT_OPTIMIZED
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:6873]: hints must be at the and of rules group or a section.
In this case - here
https://github.com/AdguardTeam/AdguardFilters/blob/f507bd5ba6f5c8472a7f77b85ecd33b3b4972318/BaseFilter/sections/specific.txt#L13889

File: AnnoyancesFilter/Popups/sections/popups_specific.txt [hunk 2]
```diff
@@ -7,6 +7,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+news.sky.com###uz_popup_container
```
Comment (Alex-302) [inline AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: What the purpose of this rule?

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 3]
```diff
@@ -9415,6 +9415,7 @@ baseendpoint.geo.de,cmp.heise.de#%#AG_onLoad(function(){if(window.location.href.
...
 krone.at#%#AG_onLoad(function(){if(-1===document.cookie.indexOf("euconsent-v2")){var c=new MutationObserver(function(){var b=document.querySelector("#didomi-notice-learn-more-button");b&&(c.disconnect(),b.click(),setTimeout(function(){var a=document.querySelectorAll("button[aria-describedby=\"didomi-purpose-BeEXA9\"]");a[1]&&a[1].click();var b=document.querySelectorAll("button[aria-describedby=\"didomi-purpose-9HxF6e\"]");b[0]&&b[0].click();var c=document.querySelectorAll("button[aria-describedby=\"didomi-purpose-JmQMZH\"]");c[0]&&c[0].click()},300),setTimeout(function(){var a=document.querySelectorAll("button[aria-describedby=\"didomi-purpose-xKLmtF\"]");a[1]&&a[1].click()},300),setTimeout(function(){var a=document.querySelector("button[aria-label*=\"speichern\"]");a&&a.click()},300))});c.observe(document,{childList:!0,subtree:!0}),setTimeout(function(){c.disconnect()},1E4)}});
 !
+news.sky.com#%#if (document.cookie.indexOf('consents') < 0) { document.cookie += 'consents=:3:1:11:10:7:8:13:12:6:5:2:9:;max-age=60' }
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:9418]: It does not fix a problem of cookies consent on my end.
Also not all platforms support JS rules. Need a universal fix.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Popups/sections/popups_specific.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: To hide a survey request.

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply BaseFilter/sections/specific.txt]: Moved.

File: AnnoyancesFilter/Popups/sections/popups_specific.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/popups_specific.txt:10]: Screen4343

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt (hunk 3 above)
Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_specific.txt]: Updated.

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt [hunk 4]
```diff
@@ -8,6 +8,10 @@
...
 @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=nintendo.*|mtv.com|ryobitools.eu|cc.com|3sat.de|zdf.de|costco.com|giantbomb.com|glovoapp.com|rte.ie|digitaltrends.com|spiele.bild.de|spiele.stern.de|vanityfair.com|saarbruecker-zeitung.de|olympics.com|dziendobry.tvn.pl|bergfreunde.de|oe24.at|epicgames.com|toteraz.pl|natgeotv.com|midomi.com|volksfreund.de|kawasaki.com|foxchannels.com.tr|ga.de|shop.mango.com|kralmuzik.com.tr|westernunion.com|roastmarket.de|new.cappex.com|vimeo.com|heute.at|ntv.com.tr|wargaming.net|metacritic.com|gq-magazine.co.uk|viafree.dk|viafree.fi|viafree.no|viafree.se|glamour.de|cbsnews.com|worldoftanks.ru|vg247.com|newyorker.com|games.miamiherald.com|eurogamer.es|eurogamer.de|bet.com|ikonpass.com|mcdelivery.de|id.trimble.com|skateboarding.transworld.net|newegg.com|docs.vmware.com|udemy.com|olx.pl|usmagazine.com|uefa.com|milwaukeetool.eu|muscleandfitness.com|20min.ch|ted.com|elisa.fi|nhl.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/135197
+!#if (!adguard_app_android && !adguard_app_mac && !adguard_app_windows && !adguard_ext_chromium && !adguard_ext_edge && !adguard_ext_firefox && !adguard_ext_safari )
```
Comment (Alex-302) [inline AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:12]: We have these platforms:
```
adguard_app_windows      -- AG for Windows
adguard_app_mac             -- AG for Mac
adguard_app_android        -- AG for Android
adguard_app_ios                -- AG for iOS
adguard_ext_chromium      -- AG browser extension for Chrome
adguard_ext_firefox          -- AG browser extension for Firefox
adguard_ext_edge             -- AG browser extension for Edge
adguard_ext_safari            -- AG browser extension for Safari
adguard_ext_opera            -- AG browser extension for Opera
adguard_ext_android_cb    -- AG content blocker for Samsung/Yandex
ext_ublock                           -- uBlock
```
Where this rule must be applied? And why it must be applied to not all platforms?

Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:12]: Hmm... I forgot to add `adguard_ext_opera`.
I think that the above rule is needed on partial AdGuard platforms that cannot apply a JS rule.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**